apps/dashboard/src/components/billing/hooks/use-subscription.ts

```diff
@@ -0,0 +1,62 @@
+import { useQuery } from '@tanstack/react-query';
+import { ApiServiceLevelEnum } from '@novu/shared';
+import { get } from '../../../api/api.client';
+import { differenceInDays, isSameDay } from 'date-fns';
+import { useMemo } from 'react';
+
+export interface ISubscriptionData {
```

Comment:
Please move this to `@novu/shared`. There should be the same interface somewhere in the API.